winsup/cygwin/pinfo.cc

```diff
@@ -1268,7 +1268,8 @@ proc_waiter (void *arg)
       if (!ReadFile (vchild.rd_proc_pipe, &buf, 1, &nb, NULL)
 	  && GetLastError () != ERROR_BROKEN_PIPE)
 	{
-	  system_printf ("error on read of child wait pipe %p, %E", vchild.rd_proc_pipe);
+	  if (GetLastError() != ERROR_OPERATION_ABORTED)
```

Comment:
yes, if we called `CancelSynchronousIo` it's because we were going to terminate the thread before, so we want it to bail out pretty much as quickly as possible.

Reply:
ok, thanks